cases/scenarios/babel/setup.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -exo pipefail
+cd "${0%/*}"
+
+source ../../cloneScenario.sh https://github.com/babel/babel.git
+
+COREPACK_ENABLE_STRICT=0 corepack yarn@1.22.21 install --ignore-scripts
```

Comment:
This script doesn't work; babel uses a newer version of yarn via corepack (but also, `yarnPath: .yarn/releases/yarn-4.0.2.cjs`), so this invocation is invalid as yarn 4 does not have `--ignore-scripts`.

This line is this way (disabling corepack script mode) in other files as the repos in existing benchmarks do not use corepack (but, the benchmarking repo does and so would throw an error). But since babel uses corepack, we can simplify to something like:

```
corepack yarn set config enableScripts false
corepack yarn install
```

However, yarn's `enableScripts` doesn't ignore workspace scripts, which isn't as secure as I want for benchmarks. No code within a cloned repo should be able to execute, and it looks like there isn't a way to enforce that after reading yarn's docs; yarn treats the workspace as being "secure" but that isn't strong enough here. https://github.com/yarnpkg/berry/issues/4780#issuecomment-1219778585

Not sure what to do about that; this is unfortunately a blocker for merging this PR. I may need to come up with some method by which we can make executing repo code safe (e.g. bwrap or something).

Reply:
I haven't tried corepack yet in Babel.
Yarn recommends installing Yarn 1 global and installing Yarn1 >= 2 in the project.
Yarn 1 will automatically use the new version of Yarn in the project.

As a simple way for Babel, we can directly remove the `postinstall`, the` postinstall` in Babel repo is only used to install Husky when developing locally.
https://github.com/babel/babel/blob/eccbd203383487f6957dcf086aa83d773691560b/scripts/postinstall.js

Reply:
Surprising is that it does not work.
It only runs while `yarn workspaces focus`.
If we are just `yarn install` it will not work.
Okay, I admit that this is a bug for Babel, because Babel also disables `enableScripts: false`.😅

Reply:
By the way, if you want to avoid any script, you may need to remove `yarn.config.cjs`.

Reply:
> By the way, if you want to avoid any script, you may need to remove `yarn.config.cjs`.

Indeed; and technically the vendored yarn itself... I'll have to think about this harder. Don't worry about changing babel at all in the meantime; this will of course be a problem for repos other than babel so I'd like to come up with a better solution.

Reply:
It may also be that I am being overly paranoid and it's not actually a problem. We could also just not have the unpinned variants, but that's not as good for long term checks.

Reply:
Oh, Babel also has its own yarn plugin and is required during the build process.

Reply:
#47 implements an install sandbox; I'll update this PR once it's deployed.

Reply:
Done. I have some other refactors but this install stuff is no longer a concern!